gradle/libs.versions.toml

```diff
@@ -6,6 +6,13 @@ managed-mailjet = "5.2.1"
 
 poi-ooxml = "5.2.3"
 kotlin = "1.7.21"
+spock = "2.3-groovy-4.0"
+testcontainers = "1.17.6"
+
+micronaut-aws = "3.9.2"
+micronaut-reactor = "3.0.0-SNAPSHOT"
+micronaut-serde = "2.0.0-SNAPSHOT"
+micronaut-views = "3.7.1"
```

Comment:
Can you raise an issue to fix this once we get a snapshot of views?

Also AWS

Reply:
#169